scripts/update.js

```diff
@@ -4,6 +4,8 @@
  */
 "use strict"
 
+require("./convert-pure-esm-to-cjs")
```

Comment:
this could unexpectedly update the `import-meta-resolve`, as we are not using package-lock.
I would suggest running a separate script to upgrade it.